Source/Core/Core/HW/EXI/EXI_DeviceEthernet.cpp

```diff
@@ -43,6 +37,40 @@ CEXIETHERNET::CEXIETHERNET()
     SConfig::GetInstance().SaveSettings();
   }
 
+  switch (type)
+  {
+  case BBADeviceType::TAP:
+    m_network_interface = std::make_unique<TAPNetworkInterface>(this);
+    INFO_LOG(SP1, "Created TAP physical network interface.");
+    break;
+  case BBADeviceType::XLINK:
+    // TODO start BBA with network link down, bring it up after "connected" response from XLink
+
+    // Perform sanity check on BBA MAC address, XLink requires the vendor OUI to be correct.
+    // Don't actually stop the BBA from initializing though
+    if (mac_addr_setting.substr(0, 8) != "00:09:bf" && mac_addr_setting.substr(0, 8) != "00:17:ab")
+    {
+      PanicAlertT("BBA MAC address %s invalid for XLink Kai. MAC address must start with 00:09:bf "
```

Comment:
Also, this was mentioned by @Techjar on IRC: those two MAC vendors are apparently the ones used by real GC hardware.

I feel we should be documenting this in some way; perhaps even in this message - and also include the fact that it is being checked for this specific reason (at least I assume thats why the check is here)